{"cells":[{"cell_type":"code","execution_count":1,"metadata":{},"outputs":[{"ename":"ImportError","evalue":"cannot import name 'OpenAI' from 'openai' (/Users/carstenlawrenz/opt/anaconda3/envs/streamlit2023/lib/python3.11/site-packages/openai/__init__.py)","output_type":"error","traceback":["\u001b[0;31m---------------------------------------------------------------------------\u001b[0m","\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)","\u001b[1;32m/Users/carstenlawrenz/GIT/SudokuSolver/ChatGPT/test.ipynb Cell 1\u001b[0m line \u001b[0;36m1\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/carstenlawrenz/GIT/SudokuSolver/ChatGPT/test.ipynb#W0sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m \u001b[39mfrom\u001b[39;00m \u001b[39mopenai\u001b[39;00m \u001b[39mimport\u001b[39;00m OpenAI\n","\u001b[0;31mImportError\u001b[0m: cannot import name 'OpenAI' from 'openai' (/Users/carstenlawrenz/opt/anaconda3/envs/streamlit2023/lib/python3.11/site-packages/openai/__init__.py)"]}],"source":["from openai import OpenAI\n"]}],"metadata":{"kernelspec":{"display_name":"streamlit2023","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.11.5"}},"nbformat":4,"nbformat_minor":2}
